binford2k-abalone/templates/config.yaml.erb

```diff
@@ -36,11 +36,11 @@
 <% if @params -%>
 :params:
 <% if @params.class == Hash -%>
-  <% @params.keys.sort.each do |key, value| -%>
+  <% @params.sort_by { |key, value|  key }.each do |key,value| -%>
   <%= key %>:
   <% if value.class == Hash -%>
-    <% if value.has_key? :map %>:map: <%= value[:map] %><% end -%>
-    <% if value.has_key? :values %>:map: <%= value[:values].inspect %><% end -%>
+    <% if value.has_key? ':map' %>:map: <%= value[':map'] %><% end -%>
```

Comment:
bah. Puppet is such a pita to manage symbols. I should just convert them to strings and switch over to using fiddyspence/hash_file.